src/main/resources/templates/dataresource.html

```diff
@@ -0,0 +1,184 @@
+											</p>
+											<p>
+												<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+												<button type="button" class="btn btn-standard">Save</button>
+											</p>
+
+										</div>
+										<div class="modal-footer">
+											<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
+						<td>Study of the topology of the IPv4 and IPv6 Internet</td>
+						<td>2015/08</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>2</td>
+						<td>DNS AMPL DDOS DEC2015 </td>
+						<td>DNS Server Spoofed Request Amplification DDoS</td>
+						<td>2015/07</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+							<td>2</td>
+							<td>SSDP DDoS attack 2015</td>
+							<td>Simple Service Discovery Protocol attack</td>
+						    <td>2015/12</td>
+							<td>
+								<button type="button" class="btn btn-standard"><span class="btn-special">Request Access</span></button>
+								<button type="button" class="btn btn-standard">Save</button>
+					</td>
+					</tr>
+
+
+					</table>
+					<hr/>
+					<div class="fa fa-share" ><strong> &nbsp;&nbsp;Link to <a href="https://www.impactcybertrust.org/"> IMPACT Data</a></strong></div>
+				</div>
+				<!--<p>Some content.</p>-->
+			</div>
+			<div id="bigdata" class="tab-pane fade">
+				<!--<h3>Menu 1</h3>-->
+				<!--<p>Some content in menu 1.</p>-->
+			</div>
+			<div id="blockchain" class="tab-pane fade">
+				<!--<h3>Menu 2</h3>-->
+				<!--<p>Some content in menu 2.</p>-->
+			</div>
+			<div id="mobile" class="tab-pane fade">
+				<!--<h3>Menu 3</h3>-->
+				<!--<p>Some content in menu 3.</p>-->
+			</div>
+		</div>
+
+
+		</div> <!--here-->
+
+	</div>
+
+
+		<div class="cc-wraper">
+			<div class="cc-contactpop fixed">
+				<h1>Do you need Help? <span>Get in touch.</span></h1>
+				<div class="renseignement">
+					<i class="fa fa-times"></i>
+					<p>We will get back to you as soon as possible. </p>
+				</div>
+				<form>
+					<ul>
+						<li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
+						<li><i class=""></i><span>Email:</span><a href="mailto:support@ncl.sg"> support@ncl.sg</a></li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)